packages/devtools_app/lib/src/charts/chart_controller.dart

```diff
@@ -414,6 +448,8 @@ class ChartController extends DisposableController
   }
 
   int normalizeTimestampIndex(int index) {
+    if (index == -1) return -1;
```

Comment:
allowing index==-1 is a bit surprising. Perhaps handle that on the caller side or make this a private method and document the contract.

Reply:
Done.